pkg/policyengine/PolicyDispatcher.go

```diff
@@ -36,8 +39,27 @@ const (
 var plog = logrus.WithField("component", "PolicyEngine")
 var MyPolicyHandler PolicyHandler
 
+type PolicyDecider interface {
+	AddLBPolicy(lbPolicy *LBPolicy) error
+	DeleteLBPolicy(lbPolicy *LBPolicy) error
+
+	AddAccessPolicy(policy *api.Policy) error
+	DeleteAccessPolicy(policy *api.Policy) error
+
+	AuthorizeAndRouteConnection(connReq *event.ConnectionRequestAttr) (event.ConnectionRequestResp, error)
+
+	AddPeer(peer *api.Peer) error
```

Comment:
These (and similar) functions cannot really fail.
Do we envision any future extension that will make them fail-able?